### DISCUSSION TITLE
Can i add multiple exporters

### DISCUSSION THREAD

**Discussion by RSoodAMAT** [open-telemetry/opentelemetry-java#3211] [Q&A]
Hi , Currently we are using otel colletor to collect all the metrices and traces . Can i explicitly use prometheus exporter along with it , and if yes then how . I dont want metrics to be collected by otel-collector but directly by prometheus exporter.

Currently we are using in below formats : 

java -javaagent:opentelemetry-javaagent-all.jar -Dotel.instrumentation.spring-batch.enabled=true -Dotel.exporter.otlp.endpoint="http://localhost:4317" -jar -Dotel.resource.attributes=service.name=importJob import/build/libs/*.jar

AND FOR DOCKER 
export OTEL_EXPORTER_OTLP_HOST="${OTEL_EXPORTER_OTLP_HOST:=xx.xxx.xx.xx}"
export OTEL_EXPORTER_OTLP_PORT="${OTEL_EXPORTER_OTLP_PORT:=4317}"

java  -javaagent:opentelemetry-javaagent-all.jar \
      -Dotel.instrumentation.spring-batch.enabled=true \
      -jar -Dspring.profiles.active=test /tc/import.jar

**Comment by jkwatson** [comment]
You can have multiple span exporters, but not metrics at the moment. This will probably be addressed in coming months.

**Reply by itsprakash84** [reply]
Yes, you can have mutiple exporters see below for Jaeger and Prometheus, As the Java opt is different you can use it. 
As mentioned by @jkwatson  mutiple metrics is not possible and also there are very limited metric at this time.

  -Dotel.traces.exporter=jaeger \
  -Dotel.exporter.jaeger.endpoint=http://tempo:14250 \
  -Dotel.metrics.exporter=prometheus \
  -Dotel.exporter.prometheus.port=9464	\
  -Dotel.exporter.prometheus.host=0.0.0.0 \
  -Dotel.instrumentation.runtime-metrics.enabled=true \
